integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java

```diff
@@ -224,7 +224,7 @@ public void testIstioModelInImage() {
 
     Path archivePath = Paths.get(ITTESTS_DIR, "../src/integration-tests/apps/testwebapp.war");
     ExecResult result = null;
-    result = DeployUtil.deployUsingRest(K8S_NODEPORT_HOST, 
+    result = DeployUtil.deployToClusterUsingRest(K8S_NODEPORT_HOST, 
```

Comment:
We usually use static import. Static import DeployUtil.deployToClusterUsingRest and change this DeployUtil.deployToClusterUsingRest to deployToClusterUsingRest.

Reply:
done